### PR TITLE
New package, calib

### PR DESCRIPTION
**Pull request by baraaorabi**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/calib/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export C_INCLUDE_PATH=${PREFIX}/include
+export LIBRARY_PATH=${PREFIX}/lib
+export CPATH=${PREFIX}/include
+
+make
```
Comment (bgruening) [inline recipes/calib/build.sh]: Is there a `make test` target that you can run?

File: recipes/calib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  patches: 
+    - Makefile.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - zlib
```
Comment (bgruening) [inline recipes/calib/meta.yaml]: zlib can be removed here

File: recipes/calib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+#    - cmake 3.12
```
Comment (bgruening) [inline recipes/calib/meta.yaml]: can you remove cmake

File: recipes/calib/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    - zlib
+  run:
+    - zlib
+test:
+  commands:
+    - calib --help
+
+about:
+  home: https://github.com/vpc-ccg/calib
+  license: MIT
```
Comment (bgruening) [inline recipes/calib/meta.yaml]: can you please add `license_file: LICENSE`

File: recipes/calib/build.sh (hunk 1 above)
Comment (dpryan79) [reply recipes/calib/build.sh]: Looks like the answer is "no".

File: recipes/calib/meta.yaml [hunk 5]
```diff
@@ -17,8 +17,6 @@ build:
 requirements:
   build:
     - make
-    - zlib
```
Comment (baraaorabi) [inline recipes/calib/meta.yaml]: Zlib is used at compile time. Shouldn't we keep this?

Comment (bgruening) [reply recipes/calib/meta.yaml]: The host section is enough. The build section is really just for the compilers and the build-machinery ... e.g. cmake etc.

Comment (dpryan79) [reply recipes/calib/meta.yaml]: No, it's in the host section, since it's a library that needs to be linked against.

**Pull request merged**